site/docs/pages/swap/swap-settings.mdx

```diff
@@ -0,0 +1,378 @@
+---
+title: <SwapSettings /> · OnchainKit
+description: Swap component slippage controls
+---
+
+import { Avatar, Name } from '@coinbase/onchainkit/identity';
+import {
+  Swap,
+  SwapAmountInput,
+  SwapButton,
+  SwapMessage,
+  SwapSettings,
+  SwapSettingsSlippageDescription,
+  SwapSettingsSlippageInput,
+  SwapSettingsSlippageTitle,
+  SwapToggleButton,
+} from '@coinbase/onchainkit/swap';
+import { Wallet, ConnectWallet } from '@coinbase/onchainkit/wallet';
+import App from '../../components/App';
+import SwapWrapper from '../../components/SwapWrapper';
+import { walletDropdownLinkCustomBaseIconSvg } from '../../components/svg/walletDropdownLinkCustomBaseIconSvg';
+
+# `SwapSettings`
+The `SwapSettings` component allows users to configure the maximum acceptable price difference (slippage) for their token swap transactions. 
+It provides a simple interface to set and adjust this tolerance level.
```

Comment:
I am a bit confused about those two senteces.

How can we make this feel a better story telling experience.

Reply:
Simplified for now prior to the story telling refactor